src/components/cart/CartSidebar/CartSidebar.module.css

```diff
@@ -0,0 +1,11 @@
+[data-variant="cart"] [data-main] {
+  @apply max-w-sm;
+}
+
+[data-variant="cart"] [data-backdrop] {
+  @apply bg-[#03044e];
```

Comment:
This is the same color used for the backdrop of the hamburger menu, so it's probably worth adding to tailwind config as a backdrop color to be reused.